test/conformance/source/environment.cpp

```diff
@@ -199,6 +201,14 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
         error = "Could not find any devices associated with the platform";
         return;
     }
+    // Get an environment variable (NUMBER_OF_DEVICES) to run tests on the appropriate number of devices
+    auto devices_num = ur_getenv("NUMBER_OF_DEVICES");
+    int count_set = devices_num ? std::stoi(devices_num.value()) : 1;
+    if (count_set <= 0 || count_set > std::numeric_limits<uint32_t>::max()) {
+        error = "Invalid NUMBER_OF_DEVICES parameter";
+        return;
+    }
+    count = std::min(count, static_cast<uint32_t>(count_set));
```

Comment:
If I won't set the env var I will always get `1`...?

Reply:
Now if you won't set the argument you will always get it on 1 device